internal/xdscache/v3/listener.go

```diff
@@ -327,6 +331,7 @@ func visitListeners(root dag.Vertex, lvc *ListenerConfig) map[string]*envoy_list
 			StreamIdleTimeout(lvc.StreamIdleTimeout).
 			MaxConnectionDuration(lvc.MaxConnectionDuration).
 			ConnectionShutdownGracePeriod(lvc.ConnectionShutdownGracePeriod).
+			AllowChunkedLength(!lvc.DisableAllowChunkedLength).
```

Comment:
Don't love the translation from disabled -> enabled, did it so the flow in the config file would be false by default rather than having to mess too much with the field being true by default, but that might be unwieldy so open to changing it

Reply:
Yeah the switch from `Disable` to `Allow` can be confusing. Maybe if we settle on using `Disable` in the config, but then translate that into `Allow` in the rest of the code, this way it's only ever one way throughout.